Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+### Personal information
 
-## Who will review the applications?
-The review panel is drawn from the OBF Board of Directors and OBF members. Any panelist with a clear conflict of interest will recuse themselves from deliberation.
+The applicants will supply personal information such as name, email id, career stage, and affiliation.
+This information will not be shared with the review panel but will be used by the fellowship chair to identify who the applicants are.
 
-## How to apply
-Fill out [the application form](https://goo.gl/forms/btbOOfkVcXVzZXxD2). Applicants will be notified of the outcome no later than 3 weeks after the application deadline.
+### Event and nature of your participation
 
-**Application Deadlines**:
+In this part, applicants will be expected to provide information about the event they have selected to participate in.
+Details that they need to provide are the name of the event, event website, dates (should take places within 1 year from the application deadline), type of the event (conference, training, or other), confirm if the event is taking place in-person or online, what is the nature of their participation (presenter, attendee or other) and how it relates to open science practices or open source software. 
 
-1 December 2019 (transition date from original three deadlines per year).
+OBF Event Fellowships are intended to enable awardees to participate in events that promote open source bioinformatics and/or open science. 
+In an open text response (maximum 250 words), applicants are asked to describe how this event and their participation in it relate to these goals.
 
-From 2020 onwards, twice a year: 1 April, and 1 October.
+### Open Source Resources
+
+OBF promotes the development of open-source bioinformatics software, documentation, or other content useful for research communities. 
+In this section, applicants are asked to provide a link to an online repository and evidence of open-source licensing. 
+
+They are further expected to describe the project that their participation in the event involves (either because they are presenting, co-developing, or because the event involves activities around this topic).
+If they cannot provide links to specific resources and licenses, then they can describe how their participation furthers open science practices.
+
+### Promoting diversity and representation
+
+One of the goals of the OBF fellowship is to increase participation from groups otherwise underrepresented in the community or at the event, including but not limited to underrepresented demographic groups (country of residence and citizenship), ethnic background (historically underrepresented and other minority groups), career stages, gender identity and expression, and people with disabilities.
```

Comment:
@nlharris does the suggestion made by Yo (and now integrated in the document) address your comment?
> One of the goals of the OBF fellowship is to increase participation from groups otherwise underrepresented at the event or in the broader community, including but not limited to underrepresented country of residence or origin, ethnic background (historically underrepresented and other minority groups), career stages, sexuality, gender identity and expression, and people with disabilities.

Reply:
@nlharris I am resolving this to see how the rendered file looks like on the website. We can come back and discuss this if needed.